open-machine-learning-jupyter-book/ml-advanced/ensemble-learning/getting-started-with-ensemble-learning.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>A link to the original article and a link to the author should be provided, referencing other chapters. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/757/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
It is a paper book, I give a amazon link